pkg/standalone/run.go

```diff
@@ -49,6 +49,7 @@ type RunConfig struct {
 	MaxConcurrency     int    `arg:"app-max-concurrency"`
 	PlacementHostAddr  string `arg:"placement-host-address"`
 	ComponentsPath     string `arg:"components-path"`
+	SpecResourcesPath  string `arg:"spec-resources-path"`
```

Comment:
getArgs() is used to get the `arg` def here and that is passed on to `daprd` executable. So if the arg is `spec-resources-path` that I believe that would break `daprd`. @hueifeng Can you verify that functionality?

Reply:
Let me try it.

Reply:
@hueifeng Were you able to check this out?

Reply:
@mukundansundar It looks like I forgot to add it to the `cmd` folder.

Reply:
@mukundansundar Add `resources-path` to the `daprd` binary https://github.com/dapr/dapr/pull/4998. After the dapr repository merge, I think it's ready for commit e2e testing.

Reply:
@mukundansundar this is now merged: https://github.com/dapr/dapr/pull/4998.